tests/tests/test-pallet-maintenance-mode.ts

```diff
@@ -0,0 +1,108 @@
+import Keyring from "@polkadot/keyring";
+import { expect } from "chai";
+import { ALITH, ALITH_PRIV_KEY } from "../util/constants";
+
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+import { createBlockWithExtrinsic } from "../util/substrate-rpc";
+
+const TEST_ACCOUNT = "0x1111111111111111111111111111111111111111";
+
+// A call from root (sudo) can make a transfer directly in pallet_evm
+// A signed call cannot make a transfer directly in pallet_evm
+
+describeDevMoonbeam("Pallet Maintenance Mode - no sudo", (context) => {
+  let events;
+  before("Try turning maintenance mode on", async () => {
+    const keyring = new Keyring({ type: "ethereum" });
+    const alith = await keyring.addFromUri(ALITH_PRIV_KEY, null, "ethereum");
+    ({ events } = await createBlockWithExtrinsic(
+      context,
+      alith,
+      context.polkadotApi.tx.maintenanceMode.enterMaintenanceMode()
+    ));
+  });
+
+  it("should fail without sudo", async function () {
+    expect(events[5].toHuman().method).to.eq("ExtrinsicFailed");
+    expect(await context.web3.eth.getBalance(TEST_ACCOUNT)).to.equal("0");
+  });
+});
+describeDevMoonbeam("Pallet Maintenance Mode - with sudo", (context) => {
+  let events;
+  before("Try turning maintenance mode on", async () => {
+    const keyring = new Keyring({ type: "ethereum" });
+    const alith = await keyring.addFromUri(ALITH_PRIV_KEY, null, "ethereum");
+
+    ({ events } = await createBlockWithExtrinsic(
+      context,
+      alith,
+      context.polkadotApi.tx.sudo.sudo(
+        context.polkadotApi.tx.maintenanceMode.enterMaintenanceMode()
+      )
```

Comment:
You have to use the council to enter maintenance mode even on moonbase https://github.com/PureStake/moonbeam/blob/master/runtime/moonbase/src/lib.rs#L1544-L1545